gensim/models/wrappers/dl4j/dl4jwrapper.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Mohit Rathore <mrmohitrathoremr@gmail.com>
+# Copyright (C) 2017 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""
+Python wrapper
+TODO: write desciption
```

Comment:
add instruction how to generate the jar file. please include pom and java files into this pr. Please add a TODO to remove the .java file after DL4J merge in your pr.